README.md

```diff
@@ -1,403 +1,42 @@
-<img width="250" src="./logo.svg" alt="Solidus logo">
+# solidus\_frontend
 
-# Solidus
+Frontend contains controllers and views implementing a storefront and cart for Solidus.
```

Comment:
I would add a notice to let new users know that this is not the recommended frontend anymore.